cmk/base/legacy_checks/raritan_pdu_plugs.py

```diff
@@ -33,12 +33,14 @@ def check_raritan_pdu_plugs(item, params, parsed):
     if data.get("outlet_name"):
         yield 0, data["outlet_name"]
 
+    required_state = params.get("required_state", params["discovered_state"])
     state, state_info = data["state"]
```

Comment:
`state` is now completely unused; even if the user did not configure anything. Is that intended? It does not seem right to me.

Reply:
> `state` is now completely unused; even if the user did not configure anything. Is that intended? It does not seem right to me.

required_state (if there is a "PDU Plug state" rule for this plug) and discovered_state are both stored as text ("on"/"off").
This is already implemented and it is what I have do deal with. Therefore I have to compare to that string using state_info and not state.

But since state and state_info have a one-to-one mapping (0,"on" | 2,"off" | 3, "unknown") it really makes no difference.

In version 1.6 a plug got a critical state when its state was different from the one from service discovery.
This could be overruled by a "State of PDU Plugs" rule.

My aim was to restore this behavior.

Reply:
@mo-ki should we include the "CRIT -" and "OK -" in the summary? This should be picked up by the respective state code right and therefore redundant?